src/Jmelosegui.Mvc.Googlemap.Examples/Views/Basic/SearchBox.cshtml

```diff
@@ -0,0 +1,119 @@
+﻿@using Jmelosegui.Mvc.GoogleMap
+
+@{
+    ViewBag.Title = "Places search box";
+    string[] libraries = { "places" };
+
+}
+<style type="text/css">
+    .controls {
+        margin-top: 10px;
+        border: 1px solid transparent;
+        border-radius: 2px 0 0 2px;
+        -webkit-box-sizing: border-box;
+        -moz-box-sizing: border-box;
+        box-sizing: border-box;
+        height: 32px;
+        outline: none;
+        box-shadow: 0 2px 6px rgba(0, 0, 0, 0.3);
+    }
+
+    #pac-input {
+        background-color: #fff;
+        font-size: 15px;
+        font-weight: 300;
+        margin-left: 12px;
+        padding: 0 11px 0 13px;
+        -ms-text-overflow: ellipsis;
+        -o-text-overflow: ellipsis;
+        text-overflow: ellipsis;
+        width: 300px;
+    }
+
+        #pac-input:focus {
+            border-color: #4d90fe;
+        }
+</style>
+
+<div style="height: 500px; border: solid 1px #cccccc">
+    <input id="pac-input" class="controls" type="text" placeholder="Search Box">
+    @(Html.GoogleMap()
+        .Name("map")
+        .Height(500)
+        .Libraries(libraries)
+        .ClientEvents(events => events.OnMapLoaded("mapEvent"))
+    )
+</div>
+
+<br />
+
+@section scripts{
+    @(Html.GoogleMap().ScriptRegistrar())
```

Comment:
The `ScriptRegistrar()` method is already called within the [_LayoutExamples.cshtml](https://github.com/jmelosegui/GooglemapMvc/blob/master/src/Jmelosegui.Mvc.Googlemap.Examples/Views/Shared/_LayoutExamples.cshtml#L17). If we include it again will cause the following error:

>You have included the Google Maps API multiple times on this page. This may cause unexpected errors.